data/sql/updates/pending_db_world/rev_1549320792563626100.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1549320792563626100');
+
+DELETE FROM `command` WHERE `name`='explorecheat';
+DELETE FROM `command` WHERE `name`='taxicheat';
+DELETE FROM `command` WHERE `name`='waterwalk';
+
+DELETE FROM `command` WHERE `name`='cheat' OR `name` LIKE 'cheat%';
+INSERT INTO `command` (`name`, `security`, `help`) VALUES 
+('cheat',           2, 'Syntax: .cheat $subcommand\r\nType .cheat to see the list of possible subcommands or .help cheat $subcommand to see info on subcommands'),
+('cheat god',       2, 'Syntax: .cheat god [on/off]\r\nEnables or disables your character''s ability to take damage.'),
```

Comment:
```suggestion
('cheat god',       2, 'Syntax: .cheat god [on/off]\r\nMakes the user invulnerable.')
```

By the way, does it work on a selected character that is NOT the GM character?

Reply:
other options can be cast on other characters though (from the description) so it might be the same here

Reply:
It doesn't work on selected characters. Only taxi, waterwalk and explore work on selected chars.